test/sign_mpe_funs.js

```diff
@@ -71,5 +92,7 @@ async function waitSignedClaimMessage(fromAccount, contractAddress, channelId, n
 
 
 module.exports.waitSignedClaimMessage        = waitSignedClaimMessage;
-module.exports.isValidSignatureClaim         = isValidSignatureClaim;
-
+module.exports.isValidSignatureClaim = isValidSignatureClaim;
```

Comment:
nitpick: the equal sign should be aligned to the former, to make it consistent with the rest of the codebase 🙏

Reply:
Will update to have consistent in the event names